activejob/test/support/integration/dummy_app_template.rb

```diff
@@ -14,7 +14,7 @@ class TestJob < ActiveJob::Base
 
   def perform(x)
     File.open(Rails.root.join("tmp/\#{x}"), "w+") do |f|
-      f.write x
+      f.write I18n.locale
```

Comment:
As long as it doesn't break other tests, this could probably work. What do you think, @cristianbica @robin850?